app/controllers/twilio_controller.rb

```diff
@@ -2,12 +2,14 @@
 require 'sanitize'
```

Comment:
Rails autoload takes care of loading those dependencies (_requires_) automatically. Could you remove them please? -- I know you didn't put them here, we're just cleaning up this controller.